modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -1034,7 +1035,45 @@ sub _summarise_website_db {
                                                               };
       }
     }
-  }
+
+    ## Get biotype definitions
+    my $terms_endpoint = $ols.'/terms';
+    # NOTE: a term url passed as a parameter to the terms endpoint
+    # should use http, lack www, and be twice(!) url-encoded
+    my $biotype_url = 'http://ensembl.org/glossary/ENSGLOSSARY_0000025';
+    my $safe_biotype_url = uri_escape(uri_escape($biotype_url));
+    my $biotype_term_url = $terms_endpoint . '/' . $safe_biotype_url;
+ 
+    sub fetch_term {
+      my %result;
+      my $url = $_[0];
+      my $term = $self->_get_rest_data($url);
+      my $children_link = $term->{_links}->{children}->{href};
+
+      $result{label} = $term->{label};
+      $result{description} = $term->{description};
+      $result{synonyms} = $term->{synonyms};
+
+      if ($children_link) {
+        $result{'children'} = [ fetch_children_terms($children_link) ];
+      }
+      return \%result;
+    }
+
+    sub fetch_children_terms {
+      my $url = @_[0];
```

Comment:
When restarting the server, perl compiler opines that "Scalar value @_[0] better written as $_[0]". I've almost given up all hope to understand perl's type annotations. I thought, `@_[0]` is pretty idiomatic. Just as `shift` is; only perhaps more explicit. Do we have any conventions regarding how to write this thing?

Reply:
Yes. You only ever use @ and % to refer to the entire array/hash (and when dereferencing). References, array slices, hash elements, etc always use $.